torch/csrc/autograd/functions/convolution.cpp

```diff
@@ -156,26 +156,57 @@ static auto view3d(const at::Tensor& tensor) -> at::Tensor {
 }
 
 static void check_input_shape_forward(const at::Tensor& input,
-				      const at::Tensor& weight,
+				      const at::Tensor& weight, const at::Tensor& bias,
 				      int64_t groups, bool transposed) {
+  int k = input.ndimension();
+
+  if (weight.ndimension() != k) {
+      std::stringstream ss;
+      ss << "Expected " << k << "-dimensional weight for " << k 
```

Comment:
it's the other way around: "Expected K-dimensional input for K-dimensional weight, but got input of size..."